exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/first_steps.ipynb

```diff
@@ -0,0 +1,820 @@
+    "    IMPORT INTO {flights_table!q} FROM CSV AT {url!s} FILE {file!s}\n",
+    "    (1 FORMAT = {date_format!s}, 2..21) \n",
+    "    SKIP = 1\n",
+    "\"\"\"\n",
+    "\n",
+    "params = {\n",
+    "    \"flights_table\": (ai_lab_config.db_schema, \"US_FLIGHTS\"),\n",
+    "    \"url\": \"https://dut5tonqye28.cloudfront.net/ai_lab/first_steps/\",\n",
+    "    \"file\": \"US_FLIGHTS_FEB_2024-fixed-booleans.csv\",\n",
+    "    \"date_format\": \"MM/DD/YYYY HH12:MI:SS AM\",\n",
+    "}\n",
+    "\n",
+    "with open_pyexasol_connection(ai_lab_config, compression=True) as conn:\n",
+    "    result = conn.execute(query, params)\n",
+    "\n",
+    "print(f\"Imported {result.rowcount()} rows.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23e99c7a-e6a2-4497-8a74-054dab3ca131",
+   "metadata": {},
+   "source": [
+    "### 3.2 Importing a Parquet File From an AWS S3 Bucket\n",
+    "\n",
+    "This demo uses \n",
+    "* a file already uploaded to S3 bucket `ai-lab-example-data-s3` \n",
+    "* and the Exasol connection object `AI_LAB_FIRST_STEPS_S3` created before.\n",
+    "\n",
+    "First we will truncate table `US_FLIGHTS`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ce8ed658-07af-433b-8d77-0aedc3e51310",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "TRUNCATE TABLE US_FLIGHTS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5dfdd7ec-5cd5-4082-a565-ee17166fd691",
+   "metadata": {},
+   "source": [
+    "**Please note**: \n",
+    "* Parquet import requires using **<span style=\"color: #40a\">Exasol version 2025.\\* or higher</span>**, see [docs.exasol.com](https://docs.exasol.com/db/latest/loading_data/load_data_parquet.htm).\n",
+    "* Hence, the import is currently commented out.\n",
+    "\n",
+    "Now we can import the Parquet file from S3 into the database:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70b4826c-12df-4b0f-a5f9-868779bc1905",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from exasol.nb_connector.connections import open_pyexasol_connection\n",
+    "\n",
+    "query = \"IMPORT INTO {table!q} FROM PARQUET AT {connection} FILE {file!s}\"\n",
+    "\n",
+    "params = {\n",
+    "    \"table\": (ai_lab_config.db_schema, \"US_FLIGHTS\"),\n",
+    "    \"connection\": \"AI_LAB_FIRST_STEPS_S3\",\n",
+    "    \"file\": \"first_steps/US_FLIGHTS_FEB_2024.parquet\",\n",
+    "}\n",
+    "\n",
+    "#with open_pyexasol_connection(ai_lab_config, compression=True) as conn:\n",
+    "#    result = conn.execute(query, params)\n",
+    "#print(f\"Imported {result.rowcount()} rows.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "61b8024f-69c1-4247-ab5a-946d5e98fe2b",
+   "metadata": {},
+   "source": [
+    "## 4. SQLAlchemy\n",
+    "\n",
+    "Function `open_sqlalchemy_connection()` returns a SQLAlchemy engine, again using the configuration from the SCS.\n",
```

Comment:
```suggestion
    "The function `open_sqlalchemy_connection()` returns a SQLAlchemy engine, again using the configuration from the SCS.\n",
```

Reply:
Question is, we should add a single sentence to explain sqlalchemy.

Reply:
See next push adding a short introduction about SQLAlchemy:

> [SQLAlchemy](https://www.sqlalchemy.org/) is a Python SQL Toolkit and Object Relational Mapper for application developers.
>                                                                
> It provides a full suite of well known enterprise-level persistence patterns, designed for efficient and high-performing database access, adapted into a simple and Pythonic domain language.
> 
> In Exasol AI Lab, the function `open_sqlalchemy_connection()` returns a SQLAlchemy engine, again using the configuration from the SCS.
> 
> This engine will be used by the examples based on SQLAlchemy.